client/src/components/NewPlayer.vue

```diff
@@ -46,7 +46,7 @@
 
         <div class="columns is-centered">
           <div class="column is-3">
-            <a class="button is-large is-fullwidth is-primary" @click="addPlayer">Add New Player</a>
+            <a class="button is-large is-fullwidth is-primary" @click="addPlayer" tabindex="0">Add New Player</a>
```

Comment:
You should instead add a `href=""` to this... that might solve the issue without the need for the tabindex

Reply:
Lets also add some WCAG compliance